Source/Cells/Text/FORMTextFieldCell.m

```diff
@@ -177,6 +177,8 @@ - (void)updateFieldWithDisabled:(BOOL)disabled {
 - (void)updateWithField:(FORMField *)field {
     [super updateWithField:field];
 
+    [self validate];
```

Comment:
Let me know if you guys see any implication of calling `validate` here. I's done to update `isValid` property considering the new value whenever the field is changed.